htdocs/admin/ticket.php

```diff
@@ -48,6 +49,55 @@
 
 $error = 0;
 
+$formSetup = new FormSetup($db);
+
+
+// Auto mark ticket read when created from backoffice
+$item = $formSetup->newItem('TICKET_AUTO_READ_WHEN_CREATED_FROM_BACKEND');
+$item->setAsYesNo();
+$item->nameText = $langs->transnoentities('TicketsAutoReadTicket');
+$item->helpText = $langs->transnoentities('TicketsAutoReadTicketHelp');
+
+// Auto assign ticket at user who created it
+$item = $formSetup->newItem('TICKET_AUTO_ASSIGN_USER_CREATE');
+$item->setAsYesNo();
+$item->nameText = $langs->transnoentities('TicketsAutoAssignTicket');
+$item->helpText = $langs->transnoentities('TicketsAutoAssignTicketHelp');
+
+// Auto notify contacts when closing the ticket
+$item = $formSetup->newItem('TICKET_NOTIFY_AT_CLOSING');
+$item->setAsYesNo();
+$item->nameText = $langs->transnoentities('TicketsAutoNotifyClose');
+$item->helpText = $langs->transnoentities('TicketsAutoNotifyCloseHelp');
+
+if (! empty($conf->product->enabled)) {
+	$item = $formSetup->newItem('TICKET_PRODUCT_CATEGORY');
+	$item->setAsCategory('product');
+	$item->nameText = $langs->transnoentities('TicketChooseProductCategory');
+	$item->helpText = $langs->transnoentities('TicketChooseProductCategoryHelp');
+}
+
+$item = $formSetup->newItem('TICKET_DELAY_BEFORE_FIRST_RESPONSE');
+$item->fieldAttr['type'] = 'number';
+$item->nameText = $langs->transnoentities('TicketsDelayBeforeFirstAnswer');
+$item->helpText = $langs->transnoentities('TicketsDelayBeforeFirstAnswerHelp');
+
+$item = $formSetup->newItem('TICKET_DELAY_SINCE_LAST_RESPONSE');
+$item->fieldAttr['type'] = 'number';
+$item->nameText = $langs->transnoentities('TicketsDelayBetweenAnswers');
+$item->helpText = $langs->transnoentities('TicketsDelayBetweenAnswersHelp');
+
+// TODO : Allow use dolibarr mail template system instead of hard coded mail template and remove this option and add a select template model option
+$item = $formSetup->newItem('TICKET_REMOVE_TRACK_URL');
+$item->setAsYesNo();
+$item->nameText = $langs->transnoentities('TicketsRemoveTrackUrl');
+$item->helpText = $langs->transnoentities('TicketsRemoveTrackUrlHelp');
+
+$item = $formSetup->newItem('TICKET_REMOVE_THIRDPARTY_MAIL_COPY');
+$item->setAsYesNo();
+$item->nameText = $langs->transnoentities('TicketsRemoveThirdPartyEmailCopy');
+$item->helpText = $langs->transnoentities('TicketsRemoveThirdPartyEmailCopyHelp');
+
```

Comment:
What about TICKET_LIMIT_VIEW_ASSIGNED_ONLY and TICKET_NOTIFY_AT_CLOSING ?

Reply:
merge mistake I guess...

Reply:
TICKET_NOTIFY_AT_CLOSING is already  present and TICKET_LIMIT_VIEW_ASSIGNED_ONLY there is only save action but form seem to not exist.